src/helpers/getPokemonImageUrl.js

```diff
@@ -0,0 +1,7 @@
+function getPokemonImageUrl() {
+  const pokemonIndex = Math.floor(Math.random() * 151) + 1;
+  const imageUrl = `https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${pokemonIndex}.png`;
+  return imageUrl;
```

Comment:
here as you are not using the imageUrl in more than one place you could just return it like:
`````
  return `https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${pokemonIndex}.png`;
`````